docs/References.bib

```diff
@@ -25,6 +25,15 @@ @article{Beckwith2011
   year={2011},
 }
 
+@book{hartle2003gravity,
```

Comment:
does this have a DOI or URL?

Reply:
This is a textbook, so it does not have a DOI or URL.

Reply:
You could link `https://www.pearson.com/us/higher-education/program/Hartle-Gravity-An-Introduction-to-Einstein-s-General-Relativity/PGM298455.html`, not entirely sure if that's useful or not

Reply:
I think it's useful because I'm sure if I search for a book entitled "Gravity" I'll find more than one result :) I think the link doesn't hurt and will potentially make someone's life easier, so a net win :)